tests/utils.py

```diff
@@ -211,14 +212,21 @@ def new_random_name(prefix):
     return f"{prefix}{suffix}"
 
 
-def create_subject_name_factory(prefix):
-    subject_index = 1
+def create_id_factory(prefix=None):
```

Comment:
Please add the types to the signature.

Also make the prefix required, we use random ids for subjects, schema fields, topics. A meaningful prefix is useful while debugging

Reply:
Also, I just noticed that all the subject schemas lost the prefix that said they were a subject, can you please add it back?

Reply:
True both. Fixing.